rotkehlchen/tests/api/test_accounting_rules.py

```diff
@@ -530,6 +536,80 @@ def test_cache_invalidation(rotkehlchen_api_server: APIServer) -> None:
     )
 
 
+@pytest.mark.parametrize('initialize_accounting_rules', [False])
+def test_manage_rules_with_event_key(rotkehlchen_api_server: 'APIServer') -> None:
+    """Test operations with accounting rules that have event_key"""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    existing_entries = add_entries(DBHistoryEvents(rotki.data.db))
+
+    rule_with_event_key: dict[str, Any] = {
```

Comment:
There is no event key here

```suggestion
   base_rule: dict[str, Any] = {
```